testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -290,8 +290,8 @@ public void testSortItemsReflectedInExplain()
     {
         // Even if the sort items are pushed down into the table scan, it should still be reflected in EXPLAIN (via ConnectorTableHandle.toString)
         @Language("RegExp") String expectedPattern = hasBehavior(SUPPORTS_TOPN_PUSHDOWN)
-                ? "sortOrder=\\[nationkey:\\w+:\\w+ DESC NULLS LAST] limit=5"
-                : "\\[5 by \\(nationkey DESC NULLS LAST\\)]";
+                ? "sortOrder=\\[(?i)nationkey(?-i):\\S+:\\S+ DESC NULLS LAST] limit=5"
+                : "\\[5 by \\((?i)nationkey(?-i) DESC NULLS LAST\\)]";
```

Comment:
don't use `(?-i)`

instead, use `(?i:nationkey)` form (please double check me)

Reply:
Yes, that works. Today I learnt more Java regex.